docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -189,21 +187,21 @@ Then run these commands:
 
 .. note:: This method has been verified with the Trusty sources in Debian Jessie and Stretch but may not be supported in earlier versions. You may want to use ``apt-get`` instead of ``apt`` in older versions.
 
-Latest Releases via Portage (Gentoo)
-++++++++++++++++++++++++++++++++++++
+Installing Ansible on Gentoo
```

Comment:
I like this a lot better than the original title. Can we mention the word "Portage" somewhere, though? In case someone is looking for the package management name? Maybe just add "You can install Ansible on Gentoo with the Portage package manager:"?